subworkflows/local/prepare_alphafold2_dbs.nf

```diff
@@ -18,96 +18,127 @@ include { COMBINE_UNIPROT   } from '../../modules/local/combine_uniprot'
 include { DOWNLOAD_PDBMMCIF } from '../../modules/local/download_pdbmmcif'
 
 workflow PREPARE_ALPHAFOLD2_DBS {
+
+    take:
+    alphafold2_db            // directory: path to alphafold2 DBs
+    full_dbs                 //  boolean: Use full databases (otherwise reduced version)
+    bfd_path                 // directory: /path/to/bfd/
+    small_bfd_path           // directory: /path/to/small_bfd/
+    alphafold2_params_path   // directory: /path/to/alphafold2/params/
+    mgnify_path              // directory: /path/to/mgnify/
+    pdb70_path               // directory: /path/to/pdb70/
+    pdb_mmcif_path           // directory: /path/to/pdb_mmcif/
+    uniref30_alphafold2_path // directory: /path/to/uniref30/alphafold2/
+    uniref90_path            // directory: /path/to/uniref90/
+    pdb_seqres_path          // directory: /path/to/pdb_seqres/
+    uniprot_path             // directory: /path/to/uniprot/
+    bfd_link                 //   string: Specifies the link to download bfd
+    small_bfd_link           //   string: Specifies the link to download small_bfd
+    alphafold2_params_link   //   string: Specifies the link to download alphafold2_params
+    mgnify_link              //   string: Specifies the link to download mgnify
+    pdb70_link               //   string: Specifies the link to download pdb70
+    pdb_mmcif_link           //   string: Specifies the link to download pdb_mmcif
+    pdb_obsolete_link        //   string: Specifies the link to download pdb_obsolete
+    uniref30_alphafold2_link //   string: Specifies the link to download uniref30_alphafold2
+    uniref90_link            //   string: Specifies the link to download uniref90
+    pdb_seqres_link          //   string: Specifies the link to download pdb_seqres
+    uniprot_sprot_link       //   string: Specifies the link to download uniprot_sprot
+    uniprot_trembl_link      //   string: Specifies the link to download uniprot_trembl
+
     main:
     ch_bfd        = Channel.empty()
     ch_small_bfd  = Channel.empty()
     ch_versions   = Channel.empty()
 
 
-    if (params.alphafold2_db) {
-        if (params.full_dbs) {
-            ch_bfd       = file( params.bfd_path )
+    if (alphafold2_db) {
+        if (full_dbs) {
+            ch_bfd       = file( bfd_path )
```

Comment:
is there a reason why you are not using `Channel.fromPath()`?

Reply:
We wanted that this channel could be use several times, now we used this syntax `Channel.value(file(foo))`, see in [ff82923](https://github.com/nf-core/proteinfold/pull/160/commits/ff82923ea57fe60f13d37704a2aef1532433a3c3) and in [b641a37](https://github.com/nf-core/proteinfold/pull/160/commits/b641a37b2a61360d7185e94c26521f3beed33015)